packages/airview-compliance-ui/src/features/application-tile/application-tile-call-to-action-button.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "@mui/material/Button";
+
+export function ApplicationTileCallToActionButton({ href, label, classNames }) {
+  const styles = useStyles();
+
+  return (
+    <Button
+      sx={styles.button}
+      className={classNames}
+      // component={Link}
+      href={href}
+      variant="contained"
+      size="small"
```

Comment:
The props API will need to allow the passing of some additional props to support routing:
- `component` prop - this should be assigned to the `component` prop of the `Button` component
- `linkProps`- this should be spread to the `Button` component

**Example:**

```jsx
export function ApplicationTileCallToActionButton({ href, label, classNames, component = "button", linkProps }) {
 return (
    <Button
      sx={styles.button}
      className={classNames}
      component={component}
      variant="contained"
      size="small"
      {...linkProps}
    >{label}</Button>
 )
}
```